.ci_support/win_cgotruecgo_varcgotarget_platformwin-64.yaml

```diff
@@ -33,9 +33,9 @@ nocgo:
 target_platform:
 - win-64
 zip_keys:
-# - - cgo
-#   - nocgo
-#   - cgo_var
+- - cgo
+  - nocgo
+  - cgo_var
```

Comment:
I'm guessing these were manually commented. Maybe there is a way we can get the intended behavior here so that re-renders still work without issues.

Reply:
man, i tried removing all the zip_vars and making things get selected based on the sole `cgo` variable. but for some reason, selecting on `[cgo]` is problematic.